packages/shared/converters/usfmToPerf.js

```diff
@@ -46,6 +46,6 @@ export const usfm2perf = (
     console.error(e);
     perf = null;
   }
-  console.log({ perf });
+  // console.log({ perf });
```

Comment:
nit: can this line be removed instead of commented?

Reply:
Sure, I was cleaning up those, I missed a few. Thanks